charts/fluentd/templates/_pod.tpl

```diff
@@ -33,23 +33,25 @@ containers:
       {{- end }}
       exec /fluentd/entrypoint.sh
   {{- end }}
-  {{- if .Values.env }}
     env:
+      - name: FLUENTD_CONF
+        value: "../../../etc/fluent/fluent.conf"
+    {{- if .Values.env }}
     {{- toYaml .Values.env | nindent 6 }}
```

Comment:
neat: ident 2

Reply:
Hi @dioguerra - happy to change this, but the indenting for this file was a bit inconsistent. So I've updated EVERYTHING to be in line as you described above. Just to make sure I got it right, I changed:

```
env:
  - name: ...
```

to

```
env:
- name: ...
```

Is that what you wanted?

Reply:
Hopefully I got this right. Also, there will be a big section of changes, I made everything in the file aligned this way so it is consistent. If you want the other stuff put back (e.g. the `containers:` section) to have two indents the way it was, just let me know. I am big on consistency, so I may have been a bit overzealous here.

Reply:
+1 for consistency. Thanks

Reply:
Can you revert this identation changes you made? and just change where you actually need to.
I can go over all identation after and make it consistent.

Reply:
```
helm template fluentd ../fluentd/ -n kube-system 
Error: YAML parse error on fluentd/templates/daemonset.yaml: error converting YAML to JSON: yaml: line 51: could not find expected ':'
```

I'm quite certain this was supposed to work out of the box.

Reply:
Made some changes, I had been testing with a local config file using `--dry-run` and that was working. I hadn't run this test you posted. 

However I made some changes as requested and the above now works (using my local copy):
```
helm template fluentd ../../helm-charts/fluent-justin/charts/fluentd -n kube-system
```

Reply:
Thanks!